cni/network/network.go

```diff
@@ -422,37 +422,38 @@ func (plugin *NetPlugin) Add(args *cniSkel.CmdArgs) error {
 
 		// Add Interfaces to result.
 		// previously just logged the default (infra) interface so this is equivalent behavior
-		cniResult := &cniTypesCurr.Result{}
+		cniResults := []cniTypesCurr.Result{}
```

Comment:
does this change anything if we get multiple interface infos in existing cases-- previously we only had 1 cni result no matter how many interface infos we had (i.e. you've considered: singularity windows delegated nic case, dual nic case, and swift linux v2 case which all can have multiple interface infos, right?)

Reply:
Yes if CNS gives us multiple interfaceInfos, containerd will get multiple stdout from us to create pods

Reply:
But *currently* I don't think that is the case in the above scenarios (**the non-backend nic** scenarios). We only return the infra in the current code (or the last interface info if no infra is found), but we claim to support multiple interface infos.

Reply:
I will make sure there is no regression here after I verify my new changes on Linux swiftv2